tests/cupy_tests/random_tests/test_permutations.py

```diff
@@ -59,7 +59,7 @@ def test_shuffle_seed2(self, dtype):
             b = cupy.array([True, True, True, False, True, False])
         else:
             a = cupy.arange(10, dtype=dtype)
-            b = cupy.array([2, 7, 5, 8, 9, 6, 0, 1, 4, 3], dtype=dtype)
+            b = cupy.array([9, 6, 0, 1, 4, 3, 2, 7, 5, 8], dtype=dtype)
```

Comment:
This test itself is not required because random behavior is not specified. It can be changed. `test_shuffle_seed1` is enough.